lib/diagnostic.rb

```diff
@@ -96,8 +122,10 @@ def eat_meat(food)
 
 ##
 # your answer, in comments, here
-#
-#
+# Using modules over direct inheritance allows your code to be more flexible
+# and easier to read. You can easily define certain methods and variables
+# and transition them over to multiple classes without having to worry
+# about other things getting mixed in.
```

Comment:
Your first point is true. Your second point is... meh. That's sorta true of both inheritance and composition. One example of an advantage that composition has over inheritance is that you can use an arbitraty number of modules as mixins, but can only inherit from one other class. And while the parent class also may inherit methods and properties from other classes, that can get really messy. Hence your first point.